src/Extension/Traits/FluentObjectTrait.php

```diff
@@ -108,6 +113,37 @@ protected function updateFluentCMSFields(FieldList $fields)
             'Locale' => 'Locale'
         ];
 
+        // Augment Localisation tab with clickable locale links to allow easy navigation between model localisations
+        if ($owner->hasExtension(CMSEditLinkExtension::class)) {
+            $controller = Controller::has_curr() ? Controller::curr() : null;
+            $request = $controller?->getRequest();
+
+            // Pass getVars separately so we can process them later
+            $params = $request?->getVars() ?? [];
```

Comment:
This is different to the previous behaviour - but I think the previous behaviour was overly cautious. If there's no request or controller somehow (which is very unlikely for this) it makes sense to just have an empty params set here.